radiotools/atmosphere/models.py

```diff
@@ -141,7 +141,8 @@ def get_vertical_height(slant_depth, model=default_model):
 
 
 def _get_vertical_height(at, model=default_model):
-    if np.shape(at) == ():
+    """ get vertical height from atmosphere, i.e., mass overburden """
+    if not hasattr(at, "__len__"):
```

Comment:
The difference is, I think, that `np.shape` only work if `at` is a numpy array? So checking for the existence of `__len__` seems better to me.